sdgym/__init__.py

```diff
@@ -4,10 +4,10 @@
 tabular data.
 """
 
-__author__ = 'MIT Data To AI Lab'
-__copyright__ = 'Copyright (c) 2018, MIT Data To AI Lab'
-__email__ = 'dailabmit@gmail.com'
-__license__ = 'MIT'
+__author__ = 'DataCebo, Inc.'
+__copyright__ = 'Copyright (c) 2022 DataCebo, Inc.'
+__email__ = 'info@sdv.dev'
+__license__ = 'BSL-1.1'
```

Comment:
I think this is not added on the other libraries, should we add it? `license` and `copyright` @npatki

Reply:
I'm curious .. where does this info ultimately show up? 

We probably should add it for all libraries, but I'd generally consider it non-blocking (pending the Q above).